%YAML 1.0
---
odometry_topic: /difodo/odometry
source_frame: openni_camera
target_frame: world
output_file_dir: /home/omar/Documents/evaluation_files/rosify_difodo

